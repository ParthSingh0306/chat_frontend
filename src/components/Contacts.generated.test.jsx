import React from "react";
import renderer, { render, screen, fireEvent } from "react-test-renderer";
import Contacts from "./Contacts";

jest.mock("../assets/logo.svg", () => "Logo");

const renderTree = (tree) => renderer.create(tree);

describe("<Contacts>", () => {
  const contacts = [
    { _id: 1, username: "User1", avatarImage: "avatar1" },
    { _id: 2, username: "User2", avatarImage: "avatar2" },
  ];
  const changeChatMock = jest.fn();
  it("should render component", () => {
    const tree = <Contacts contacts={contacts} changeChat={changeChatMock} />;
    expect(renderTree(tree).toJSON()).toMatchSnapshot();
  });
});
