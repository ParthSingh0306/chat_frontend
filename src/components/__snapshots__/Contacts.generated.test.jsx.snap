// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Contacts> should render component 1`] = `null`;
